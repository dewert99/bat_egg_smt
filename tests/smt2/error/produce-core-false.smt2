(set-option :produce-unsat-cores false)
(declare-const a Bool)
(assert (! a :named an))
(check-sat-assuming ((not an)))
(get-unsat-core)