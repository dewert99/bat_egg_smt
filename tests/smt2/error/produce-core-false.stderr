The option `:produce-unsat-cores` must be set to true at 5:2
(get-unsat-core)
 ^

